sql-files/upgrades/2013-03-09--01-56.sql

```diff
@@ -18,6 +18,6 @@
 -- You should have received a copy of the GNU General Public License
 -- along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-ALTER TABLE `login` ADD COLUMN `pincode` VARCHAR(4) NOT NULL DEFAULT '';
-ALTER TABLE `login` ADD COLUMN `pincode_change` INT(11) unsigned NOT NULL DEFAULT '0';
+ALTER TABLE `login` ADD COLUMN `pincode` VARCHAR NOT NULL DEFAULT '';
```

Comment:
`pincode` is a `VARCHAR` and should keep it's column size.

Reply:
good eyes